autoload/tagbar.vim

```diff
@@ -354,6 +354,7 @@ function! s:CreateAutocommands() abort
         if exists('##QuitPre')
             autocmd QuitPre * let s:vim_quitting = 1
         endif
+        " ������Զ��������ڸ��˻��������ò���, �����׵��´��ļ�ʱ��Ļ�������ˢ��,���Թر�
```

Comment:
Please add comments in English (even if it's Google Translate) so the rest of us can figure out what this is about.